src/SearchInput/SearchInput.tsx

```diff
@@ -118,9 +118,20 @@ export const SearchInput = forwardRef<HTMLInputElement, SearchInputProps>(
     const [searchQuery, setSearchQuery] = useState<string | null>(null)
     const [highlightedIndex, setHighlightedIndex] = useState(-1)
 
+    const handleBlur = () => {
```

Comment:
I think both these paths do the same thing?

Reply:
🙈 - fixed it